tests/dummy/app/templates/application.hbs

```diff
@@ -22,7 +22,13 @@
       <p>Since there is no way to automate screen readers for testing, this is an interactive script to help you run all of the tests inside of each screen reader.</p>
 
       <h2>Tests</h2>
-      <p>Each of these tests should be reviewed in every single OS/AT/Browser combination.</p>
+      <p>Each of these tests should be reviewed in the appropriate OS/AT/Browser combination. Common browser/AT combinations include:</p>
```

Comment:
The only issue I have with this is that there is already a common misconception out there that any OS/AT combination should be expected to work. As in, it ends up being a business requirement. I.E, if you fire up NVDA, it should work with Chrome. It doesn't, NVDA has said as much (unless this has recently changed).